core/templates/dev/head/components/question-directives/questions-list/questions-list.directive.ts

```diff
@@ -190,25 +190,52 @@ oppia.directive('questionsList', [
               controller: [
                 '$scope', '$uibModalInstance',
                 function($scope, $uibModalInstance) {
-                  $scope.selectedSkillIds = [];
+                  var fillSkillDifficulties = function(length) {
```

Comment:
Maybe call this initializeSkillDifficulties() or init() (and do all the initialization in here)?

Reply:
Done